app/controllers/admin/communities_controller.rb

```diff
@@ -54,17 +54,25 @@ def edit_welcome_email
   end
 
   def create_sender_address
-    # TODO validate email
-
-    EmailService::API::Api.addresses.create(
+    res = EmailService::API::Api.addresses.create(
       community_id: @current_community.id,
       address: {
         name: params[:name],
         email: params[:email]
       })
 
-    flash[:notice] = t("admin.communities.outgoing_email.successfully_saved")
-    redirect_to action: :edit_welcome_email
+    if res.success
+      flash[:notice] =
+        t("admin.communities.outgoing_email.successfully_saved")
+
+      redirect_to action: :edit_welcome_email
+    else
+      flash[:error] =
+        "An error occured while saving address: #{res.error_msg}" # Shouldn't come here.
```

Comment:
My feeling is that we should just always use translatable strings even if it's unlikely that these things happen. That way we know that we at least can translate these if needed and the extra work using localized str is nothing.